src/view-mixin.js

```diff
@@ -208,26 +188,29 @@ var AbstractView = Backbone.View.extend({
 
     this.triggerMethod.apply(this, ['before:destroy'].concat(args));
 
-    // mark as destroyed before doing the actual destroy, to
-    // prevent infinite loops within "destroy" event handlers
-    // that are trying to destroy other views
+    // update lifecycle flags
     this._isDestroyed = true;
-    this.triggerMethod.apply(this, ['destroy'].concat(args));
+    this._isRendered = false;
 
     // unbind UI elements
     this.unbindUIElements();
 
-    this._isRendered = false;
-
     // remove the view from the DOM
-    this.remove();
+    https://github.com/jashkenas/backbone/blob/1.2.3/backbone.js#L1235
```

Comment:
destroy event was moved to the end of this call, which in general makes more sense, but this required breaking up `Backbone#remove` to move `stopListening` until after the trigger.  Although `_removeElement` is _'d it is documented as overrideable, and Bb isn't semver anyways, so this seems ok.